acceptance-tests/src/main/kotlin/org/ostelco/at/common/StripePayment.kt

```diff
@@ -83,7 +83,7 @@ object StripePayment {
      * verify that the correspondng 'setDefaultSource' API works as
      * intended.
      */
-    fun getDefaultSourceForCustomer(stripeCustomerId: String) : String {
+    fun getDefaultSourceForCustomer(customerId: String) : String {
```

Comment:
Makes sense.  I renamed it from `customerId` to `stripeCustomerId` when they were different.